api-samples/identity/index.html

```diff
@@ -0,0 +1,32 @@
+<html>
+  <head>
+    <title>Identity API Sample App</title>
+    <link rel="stylesheet" type="text/css" href="sample_support/standard.css">
+    <script src="identity.js"></script>
+    <!-- Disabling logger -->
+    <!-- <script src="sample_support/sample_support.js"></script> -->
```

Comment:
This seems to be old boilerplate from the apps code. Do you think it makes sense to remove this and the `sample_support` folder entirely? The only thing we seem to lose from that is some styling, so perhaps we can copy over the relevant bits.